src/form/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import React, {
+  FC,
+  TextareaHTMLAttributes,
+  useLayoutEffect,
+  useRef
+} from 'react';
+
+import css from './Textarea.module.css';
+
+export interface TextareaProps
+  extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+  /**
+   * Mark field as errored
+   */
+  error?: boolean;
+
+  /**
+   * If true, the field will take up the full width of its container.
+   */
+  fullWidth?: boolean;
+
+  /**
+   * Size of the field.
+   */
+  size?: 'small' | 'medium' | 'large';
```

Comment:
@steppy452 - Do u think we should also adjust fonts w/ this?

Reply:
I think no. At least in other UI libraries (MUI) font is still the same

Reply:
no, i don't think i've ever seen the font sizes change with the field size